api/views/CloudRecordSummaryView.py

```diff
@@ -272,38 +274,10 @@ def _request_to_token(self, request):
                               "but not of expected form.")
             self.logger.error(request.META['HTTP_AUTHORIZATION'])
             return None
-        self.logger.info("Authenticated: %s...", token[:15])
-        self.logger.debug("Full token: %s", token)
+        self.logger.info("Successfully extracted Token")
```

Comment:
Probably overkill, but you can make which message is logged conditional on the log level, so either log the basic message or the one with full details. Mostly a choice to ponder for the future.

Reply:
yeah.... I did um and ah over how to log this bit.